tensorflow/compiler/xla/g3doc/operation_semantics.md

```diff
@@ -1382,6 +1382,10 @@ For a more intuitive description, see the "Informal Description" section below.
 | `indices_are_sorted`   | `bool`              | Whether the indices are       |
 :                        :                     : guaranteed to be sorted by    :
 :                        :                     : the caller.                   :
+| `use_atomic`           | `bool`              | Whether to use atomic         |
```

Comment:
Instead of phrasing this as an implementation property (use atomics or not) we should phrase this as a semantic property (indices are duplicated or not).

We should also spec what happens if the precondition is not satisfied.  I.e.if `unique_indices` is true but there are duplicate indices.  We'd prefer to avoid UB in XLA, but producing a garbage result in such cases is fine.

Reply:
Ok. I'll do this change everywhere.

Reply:
The rename is done.